language/stdlib/src/main.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use std::fs::File;
+use std::io::Write;
+use std::path::{Path, PathBuf};
+use stdlib::{
+    build_stdlib, compile_script, filter_move_files, STAGED_EXTENSION, STAGED_OUTPUT_PATH,
+    STAGED_STDLIB_NAME, TRANSACTION_SCRIPTS,
+};
+
+fn main() {
```

Comment:
Maybe a comment about what this is intended to do? Or is there a README somewhere?

Reply:
I think adding a comment is probably a good idea. I'll add it in to this PR :)